poetry.lock

```diff
@@ -1,38 +1,64 @@
+# This file is automatically @generated by Poetry 1.4.1 and should not be changed by hand.
```

Comment:
Given the version of Poetry v1.4.1 is required for this lock file - please can we make sure this is captured somewhere in either pyproject.toml or the readme.